src/datatable/expr/reduce.py

```diff
@@ -104,10 +109,10 @@ def corr(col1, col2):
 
 
 # noinspection PyShadowingBuiltins
-def sum(iterable, start=0):
+def sum(iterable=None, start=0):
```

Comment:
I'm not sure why we need this change. Also, this PR is for the count function, so I propose we don't change others to make it clear.